### DISCUSSION TITLE
Stereo Tool vs. master_me

### DISCUSSION THREAD

**Discussion by wppronto** [AzuraCast/AzuraCast#6997] [Q&A]
I'm curious how many people are using Stereo Tool vs. master_me vs. other options. I've tested and found Stereo Tool to be very powerful, but very hard to dial in the right settings. master_me seems great but I'm not sure I'm tweaking the settings right (custom Liquidsoap configuration).

**Comment by RadioHerzblut** [comment]
If you use STEREO TOOL you do not need MASTER_ME.

This is how my sound sounds with Stereo Tool and Azuracast:

https://laut.fm/radioherzblut

You can find more basic settings here. I then customised it to my needs 👍 
https://tdcat.com/stereotool

**Reply by wppronto** [reply]
In case it helps anyone who has been searching for great Stereo Tool presets, after trying a lot I think I am also going with the settings from TDCat and then tweaking a bit. His settings bring out lots of great nuances and handle compression without sounding artificial.

**Comment by JeffersonStarseed** [comment]
I finally got the paid version of StereoTool Advanced Dynamics, and I have it installed and have been using it on my radio server.  Like yourself wppronto and RadioHerxblut, I tried a LOT of the preset/configs and was unhappy with most of them.  They sound professional, but they always seem to "overprocess" the music, too much compression, lack of gain (as compared to other stations or even YouTube), or just something that just don't sound right.  Like yourself I stumbled onto TDCat's presets and liked them the best, ...then started a very slight tweaking process on EQ, Quality, and gain and made my own edited preset.  I'm finally happy with the end-result I was able to get, although it's not perfect.

As far as Azuracast radio servers my DJ friends have started, they all love the Master_me option built-into Azuracast with the General Music setting, and 'Target Loudness' set to either 13 or 12 LUFS.  I have more stations being set-up (including my own)on our server, but DJ Jesse's station is up and running with StereoTool at:

https://www.starseedentertainment.net/starseed-radio

**Comment by wppronto** [comment]
Great job! Your station sounds great, programming-wise and sound quality-wise! I miss Germany! I used to visit Aachen a lot on business.

I've also settled on TDCat's presets, but I haven't been as successful as you in fine-tuning them. Are you willing to share your preset and/or any guidance? For example, which TDCat settings did you start with? I like them all, but I've leaned toward the "Final" version.

**Reply by JeffersonStarseed** [reply]
I'm willing to share it.  I renamed it.  I'm not too sure on how much different it is, but I tweaked it from the TDcat Final version.  But it won't let me attach a .sts file here on the reply.

**Comment by Moonbase59** [comment]
The problem with tools like _Stereo Tool_, _master_me_ or even SAM settings is that they offer _a lot_ of settings, made by professionals for professionals. They can greatly enhance some aspects of a good playout chain, but in 90% of all cases, used wrongly, they simply _ruin_ the sound.

With the possibilities you get to today with software, you’d often need to be an audio engineer (or hire one) to achieve the best results. This is especially true when a piece of such software comes with presets not suitable for a desired use case. It’s super-easy to "turn the wrong knob" and never find your way back to a good sound.

Additionally, the goals propagated today, since the start of the "loudness wars", are simply _the wrong ones_, in my opinion. Unfortunately, listening habits have adapted, since "the big ones" like many radio stations, CD producers, YouTube, Spotify, etc. all play for "loudest station on the dial". Dynamics in audio keep vanishing, the loudest (often clipping) noise "wins". A shame, really.

My advice for stations I consult since many years is always to stand out by trying to go for the _better_ sound, and consistent packaging, not the loudest or the one with most effects. Listener fatigue is reduced, and the discerning listener will _love_ it.

It may be unpopular, but I also still advise to use loudness targets of -23 LUFS for video, and -18 LUFS for radio. _Not_ any (in my opinion) idiotic stuff like "I 'gain' all my files at 93 dB with MP3Gain" (without even knowing what loudness, dB, LU, dB SPL _are_), or even -16 LUFS, -14 LUFS or any other "supposed-to-be-louder" targets.

That said, both _master_me_ and _Stereo Tool_ are _excellent tools_—if you know what you’re doing. But both cannot "magically" make a good station sound out of shit. Garbage in—garbage out, as an old saying goes. You can make an outstanding station without any of these, using high-quality audio and a carefully designed processing and playout chain.

**Comment by wppronto** [comment]
Agreed. That reminds me of wise words given to me by Joseph Watt, an amazing / brilliant / superb remixer (of Razormaid Records). I had a similar record label in the 1980s (a DJ remix service). Joseph had a magical way of multi-band EQing that brought out magic in any song. I asked him what the secret was. He said, "It's really simple. I just go through all the sliders, play with each one, and ask myself 'How much of this sound do I want in my record?'" Of course, it was only that easy for him...not me. :-)

**Comment by Moonbase59** [comment]
I still remember fondly all the good advice I got from a real brilliant audio engineer (Klaus from Cologne). Without him I’d never understood the basics, even. Sadly, he vanished in the 1980’s, and no one has ever heard of him since.

**Comment by JeffersonStarseed** [comment]
Agreed on that one.  The majority of hobby DJ's usually have 128 kb MP3 files, most of which sound horrible anyway, and then they try to compensate with gain and EQ.  Hell, I'm still trying to replace my own MP3 files I have with better-recorded and better format files.  With the front end now fixed & settled, I'm trying to work on the back-end "Source" media & songs.  Like you said Moonbase, garbage in garbage out.  Also, I've been using some nice basic free plug-ins from TDR (Tokyo Dawn Records) for EQ and comp/limiting (that in my opinion are every bit as good or better than FabFilter) as well as the YouLean loudness meter at the back-end before stream-out.

In all seriousness though, after playing with StereoTool for an hour.... I love the end-result I got, but for awhile there I thought I was going to end up pulling all my hair out.  LOL.  I learned **_not to mess_** with the compression/limiting & other settings, and focus more on trying to taylor the 'overall' EQ or tone ...and to let the StereoTool, and the selected Preset, do the work for me that I paid for.

That's the one great feature of the Master_me plug-in for my friends using Azuracast.  It is pretty straightforward and easy to set in seconds, only having to choose the media-setting and LUF target.

**Comment by wppronto** [comment]
I like the quality from I Like Music and 7Digital, but that comes with a price.

**Comment by Moonbase59** [comment]
I have a rather stringent pre-processing scheme: 11 steps before a song goes into the main database, including loudgain, BPM calculation, fingerprinting and lots of special tagging. Only these are allowed for broadcasting.

Having a collection of roughly 160k songs, collected over decades, unfortunately some are still MP3 (320k mostly), but everything new is either a CD FLAC rip, or bought FLAC/WAV material. Fortunately, more and more shops offer FLAC nowadays. :+1:

**Comment by wppronto** [comment]
Thanks for your willingness to share! I think GitHub will accept .zip files?

Otherwise, if you can send it to mark [at] the baileys [dot] com, I’d appreciate it. I’m happy to keep trying to learn and share anything good I come up with (if anything).

Thanks,

Mark

**Reply by JeffersonStarseed** [reply]
Here you go (zipped).
[TST Station Config 1.zip](https://github.com/AzuraCast/AzuraCast/files/14729664/TST.Station.Config.1.zip)
